.travis.yml

```diff
@@ -14,5 +14,5 @@ script:
 - cd docs && make html
 env:
   global:
-  - secure: hOHQ0Tqh7h2VlAJswYcRV7YV2HDxnwaq0Sdom61/qlF+oyxaMTSuYGJdGG+981DHyYFkxLX97yWcvs39UlpgWBCJN7obkdgy6kR3FZDraAVdgrBAEBYjE9fhla0Mw4PgCxr4t1dI82iUkg5u0CPF26a1ZgqlCvcskfNnjnXJXfY=
-  - secure: RQn/pmT+4MpB2aeL+63XwshftbSm+j28d/diHnUqUhr/JKXwoV9xoRLgZmTHJQzMVuaCYxaFxjfJsimVPxX1GZJm6J6vl6suNDve8GohirrRXjexmqqnQW3f+0btieOQjXmawI9YVd9m5D8t0+twE7r8kRExAztt2lHM9SFQ+Qg=
+  - secure: QktJb8B+STvIhcKu4oCG2f0W5IqzHkQVPP4uE5nxCZw64tKTRGhiHmtofxRMDqKl6ORlc9dSn8yHpyR4kCwDXSGYiOJi3jnI2bKz7DoqiKry5y+/PaUvJhYuVKfirTr2F5Wwzi1c06Zm7jQ1EpRqMe8qiZaqw1G5JGRDwnpqzoA=
+  - secure: V5HoVHPokoTRfpZstIn/O542w4dStXnAUwxSMTl1EzyaBMqONvBiW28Xkbxh5XekLa3NumoZKh9ZZQ7WwNXO00pmieNeYLXcWByztF937ozsraBHhmTnVvK59O4IkxlVy/jlxHrtIcRMdzXuIyubOppzYVUSc/rP+sbXXI4314Y=
```

Comment:
Nope! This is fine! It's generated by the command `travis encrypt AWS_ACCESS_KEY_ID=xxx --add`, which uses the Travis ruby gem to set encrypted credentials as environment variables. In yaml it's valid without the quotes.